assessmentModel/src/commonMain/kotlin/northwestern/MtbAssessment.kt

```diff
@@ -0,0 +1,223 @@
+        FlankerInstructionStep::class with FlankerInstructionStep.serializer()
+        FlankerForm::class with FlankerForm.serializer()
+        //FlankerParentForm::class with FlankerParentForm.serializer()
+        FlankerInstructionForm::class with FlankerInstructionForm.serializer()
+        IntIntChoiceInputField::class with IntIntChoiceInputField.serializer()
+    }
+    polymorphic(Assessment::class) {
+        MtbAssessmentObject::class with MtbAssessmentObject.serializer()
+    }
+    polymorphic(Question::class) {
+
+    }
+}
+
+val flankerStringIntSerializersModule = SerializersModule {
+    polymorphic(Node::class) {
+        StringIntChoiceInputField::class with StringIntChoiceInputField.serializer()
+    }
+}
+val flankerIntIntSerializersModule = SerializersModule {
+    polymorphic(Node::class) {
+        IntIntChoiceInputField::class with IntIntChoiceInputField.serializer()
+    }
+}
+val flankerDoubleIntSerializersModule = SerializersModule {
+    polymorphic(Node::class) {
+        DoubleIntChoiceInputField::class with DoubleIntChoiceInputField.serializer()
+    }
+}
+
+@Serializable
+@SerialName("Flanker")
+data class MtbAssessmentObject(
+    override val identifier: String,
+    @SerialName("steps")
+    override val children: List<Node>,
+    override val versionString: String? = null,
+    override val resultIdentifier: String? = null,
+    override var estimatedMinutes: Int = 0,
+    val taskOrientation: String = "portrait",
+    val defaultSteps: MtbDefaultSteps? = null,
+    val stepRules: List<MtbStepRule> = listOf()
+
+
+) : NodeContainerObject(), Assessment {
+    override fun createResult(): AssessmentResult = super<Assessment>.createResult()
+    override fun unpack(fileLoader: FileLoader, resourceInfo: ResourceInfo, jsonCoder: Json): MtbAssessmentObject {
+        imageInfo?.copyResourceInfo(resourceInfo)
+        val copyChildren = children.map { it.unpack(fileLoader, resourceInfo, jsonCoder) }
+        val copy = copy(children = copyChildren)
+        copy.copyFrom(this)
+        return copy
+    }
+}
+
+@Serializable
+data class MtbDefaultSteps(
+    val numberOfSteps: Int,
+    val startingStepGroup: String
+)
+
+//@Serializable
+//@SerialName("flankerParentForm")
+//data class FlankerParentForm(
+//    override val identifier: String,
+//    val isPractice: Boolean = false,
+//    @SerialName("steps")
+//    val children: List<Node> = listOf()
+//) : MtbStep()
+
+@Serializable
+@SerialName("flankerForm")
+data class FlankerForm(
+    override val identifier: String,
+    val isPractice: Boolean = false,
+    @SerialName("inputFields")
+    override val children: List<Node> = listOf(),
+    //Could remove this and use above "flankerParentForm" instead
+    val steps: List<Node> = listOf()
+) : MtbStep(), FormStep
+
+
+@Serializable
+@SerialName("flankerInstructionForm")
+data class FlankerInstructionForm(
+    override val identifier: String,
+    val isInstruction: Boolean = true,
```

Comment:
Nit: Things like `isInstruction` and `isPractice` are very "Swift-like" syntax. Is there a better word choice on Kotlin?

For example, I moved away from `isExclusive` and `isOptional` to `exclusive` and `optional` to match the Json coming from Alx's POJO objects.